fhir-persistence-blob/src/main/java/com/ibm/fhir/persistence/blob/BlobStorePayload.java

```diff
@@ -1,27 +1,44 @@
 /*
- * (C) Copyright IBM Corp. 2021
+ * (C) Copyright IBM Corp. 2021, 2022
  *
  * SPDX-License-Identifier: Apache-2.0
  */
  
 package com.ibm.fhir.persistence.blob;
 
-import com.azure.core.http.rest.Response;
-import com.azure.core.util.BinaryData;
-import com.azure.storage.blob.BlobClient;
-import com.azure.storage.blob.models.BlockBlobItem;
-import com.azure.storage.blob.options.BlobParallelUploadOptions;
+import java.util.concurrent.CompletableFuture;
+import java.util.logging.Logger;
+
+import com.azure.storage.blob.BlobAsyncClient;
+import com.azure.storage.blob.specialized.AppendBlobAsyncClient;
 import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+import com.ibm.fhir.persistence.payload.PayloadPersistenceResult;
 import com.ibm.fhir.persistence.util.InputOutputByteStream;
 
+import reactor.core.publisher.Flux;
+
 /**
  * DAO command to store the configured payload in the Azure blob
  */
 public class BlobStorePayload {
+    private static final Logger logger = Logger.getLogger(BlobStorePayload.class.getName());
+    
+    // The maximum size of a single append blob block call
+    private static final int MAX_APPEND_BLOB_BLOCK_SIZE = AppendBlobAsyncClient.MAX_APPEND_BLOCK_BYTES;
```

Comment:
I looked this up and it looks to be 4MB currently

Reply:
I've added a note on this limitation to the README.